lib/node_modules/@stdlib/array/complex64/docs/types/index.d.ts

```diff
@@ -195,6 +195,50 @@ type TernaryMapFcn<U> = ( this: U, value: Complex64, index: number, arr: Complex
 */
 type MapFcn<U> = NullaryMapFcn<U> | UnaryMapFcn<U> | BinaryMapFcn<U> | TernaryMapFcn<U>;
 
+/**
+* Reducer Function Invoked for each element in an array.
+*
+* @returns accumulated result
+*/
+type NullaryReducer<U> = ( this: U ) => ComplexLike;
+
+/**
+* Reducer Function Invoked for each element in an array.
+*
+* @param acc - accumulated result
+* @returns accumulated result
+*/
+type UnaryReducer<U> = ( this: U, acc: ComplexLike ) => ComplexLike;
+
+/**
+* Reducer Function Invoked for each element in an array.
+*
+* @param acc - accumulated result
+* @param value - current array element
+* @returns accumulated result
+*/
+type BinaryReducer<U> = ( this: U, acc: ComplexLike, value: Complex64 ) => ComplexLike;
+
+/**
+* Reducer Function Invoked for each element in an array.
+*
+* @param acc - accumulated result
+* @param value - current array element
+* @param index - current array element index
+* @returns accumulated result
+*/
+type TernaryReducer<U> = ( this: U, acc: ComplexLike, value: Complex64, index: number ) => ComplexLike;
+
+/**
+* Reducer Function Invoked for each element in an array.
+*
+* @param acc - accumulated result
+* @param value - current array element
+* @param index - current array element index
+* @returns accumulated result
+*/
+type ReducerFn = NullaryReducer<U> | UnaryReducer<U> | BinaryReducer<U> | TernaryReducer<U>;
```

Comment:
You're missing a `QuaternaryReducer` where the last argument is the array on which `reduce` is invoked.